business/workloads.go

```diff
@@ -2034,11 +2124,21 @@ func (in *WorkloadService) streamParsedLogs(cluster, namespace, name string, opt
 			break
 		}
 
-		entry := parseLogLine(line, opts.IsProxy, engardeParser)
```

Comment:
I think maybe this PR could be greatly simplified by providing a new regex pattern to this line:
```
engardeParser := parser.New(parser.IstioProxyAccessLogsPattern)
```
Something like: `parser.New(parser.ZTunnelLogsPattern)`?

Reply:
Well, this is really from an external library so, we should create a parser and a parse function in our code, as this library doesn't include a ztunnel parser yet.

Reply:
Ah for some reason I didn't realize this was from an external library.

Reply:
But right maybe would be better to try to use a regex or similar

Reply:
I've change the line processing, hopefully this approach would be more efficient